var/spack/repos/builtin/packages/pmt/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pmt(CMakePackage):
```

Comment:
Any reason not to call this parallelmergetree?

Reply:
Not really, just to make it short. For consistency with the repository name we could rename it to parallelmergetree, @cyrush what do you think (this change has to be reflected also in the Ascent uberenv variant [dependencies](https://github.com/Alpine-DAV/ascent/blob/develop/scripts/uberenv/packages/ascent/package.py#L120))?

Reply:
@spetruzza -- I am fine with this change, I'll make it when I rebase now that babelflow is in. The uberenv path will be different for a bit until we can cut over to using new spack, so we will have to be mindful of that.